### PR TITLE
feat(ui): update landing page hero section

### PR DESCRIPTION
**Pull request by matthewvogas**
### Summary
This PR implements the hero section of the landing page as per the Figma design. The hero section includes the following key elements:

- Responsive layout for desktop and mobile views.
- Light and Dark Mode
- Call-to-action button to drive user engagement.
- Branding Match.

### Linked Issues:
Closes https://github.com/holdex/holdex-venture-studio/issues/625 (Problem: Hero Section Design Not Implemented Yet)
References https://github.com/holdex/marketing/issues/68 (Goal:  Update Holdex landing page)

eta: 16

## Summary by CodeRabbit

- **New Features**
	- Introduced reusable Pug mixins for enhanced page layout and responsiveness.
	- Added a visually dynamic hero section, improving the overall presentation.
	- Implemented new components for displaying key performance indicators and client logos.
	- Enhanced call-to-action section to boost user engagement.

- **Style Enhancements**
	- Added new background image styles for more visually appealing design options.

### CODE REVIEW

**Review by markholdex — CHANGES_REQUESTED** [review]
@matthewvogas 

1. Logo loop is not infinite.
2. The metric sections doesn't match design. 
3. Heading title is outside the container.

Please be careful and make sure you test your own work. Thanks!

**Comment by matthewvogas** [comment]
@markholdex 

1. Logo loop is not infinite.
2. The metric sections doesn't match design.
3. Heading title is outside the container.

Issues solved, also refactored to have less code and better maintainability

**Comment by markholdex** [comment]
@matthewvogas much better. I will provide the updated copy and we'll merge after that.

**Comment by matthewvogas** [comment]
> @matthewvogas much better. I will provide the updated copy and we'll merge after that.

Screenshot 2024-09-11 at 10 56 37 PM

Let me know which of the two, the last ones you left in comments, or the ones on the right.

**Comment by matthewvogas** [comment]
> @matthewvogas much better. I will provide the updated copy and we'll merge after that.

Screenshot 2024-09-11 at 10 56 37 PM

Let me know which of the two, the last ones you left in comments, or the ones on the right.

**Comment by zolotokrylin** [comment]
@markholdex @matthewvogas I suggest we finalize this PR, but I have a few questions: 
1. is the mobile version missing in this PR?
2. where did the BTC chart and `Web3 is not just a financial movement; it’s an evolution of the internet.` quote go?

File: src/routes/(pages)/template.pug [hunk 1]
```diff
@@ -1,9 +1,33 @@
...
+      +wrap-subcontainer("$1,500M+", "Total value locked")
+      +wrap-subcontainer("$900M+", "Market Cap")
+      +wrap-subcontainer("+100,000", "Concurrent users")
+
+    +wrap-light
+      +wrap-subcontainer("$1,500M+", "Total value locked")
+      +wrap-subcontainer("$900M+", "Market Cap")
+      +wrap-subcontainer("+100,000", "Concurrent users")
+
+    +cta('The results of ', 'our incubated projects', ' speak for themselves', '/portfolio')
```
Comment (zolotokrylin) [inline src/routes/(pages)/template.pug:26]:
```suggestion
    +cta('The total results of ', 'our portfolio companies', ' speak for themselves', '/portfolio')
```

File: src/routes/(pages)/template.pug [hunk 2]
```diff
@@ -1,9 +1,33 @@
...
+    +wrap-dark
+      +wrap-subcontainer("$900M+", "Combined Valuation")
+      +wrap-subcontainer("$1B", "Transaction Volume")
+      +wrap-subcontainer("100K", "Users Supported")
+
+    +wrap-light
+      +wrap-subcontainer("$900M+", "Combined Valuation")
+      +wrap-subcontainer("$1B", "Transaction Volume")
+      +wrap-subcontainer("100K", "Users Supported")
+
```
[the remark this answers is not in the document]
Comment (zolotokrylin) [reply src/routes/(pages)/template.pug]: @matthewvogas this comment makes sense.

File: src/routes/(pages)/template.pug [hunk 3]
```diff
@@ -1,9 +1,33 @@
...
+  +section-kpis-wrapper
+    +wrap-dark
+      +wrap-subcontainer("$1,500M+", "Total value locked")
+      +wrap-subcontainer("$900M+", "Market Cap")
+      +wrap-subcontainer("+100,000", "Concurrent users")
+
+    +wrap-light
+      +wrap-subcontainer("$1,500M+", "Total value locked")
+      +wrap-subcontainer("$900M+", "Market Cap")
+      +wrap-subcontainer("+100,000", "Concurrent users")
```
Comment (zolotokrylin) [inline src/routes/(pages)/template.pug:24]:
```suggestion
      +wrap-subcontainer("100,000+", "Users served")
```

File: src/routes/(pages)/template.pug [hunk 4]
```diff
@@ -1,9 +1,33 @@
...
+
+  +section-kpis-wrapper
+    +wrap-dark
+      +wrap-subcontainer("$1,500M+", "Total value locked")
+      +wrap-subcontainer("$900M+", "Market Cap")
+      +wrap-subcontainer("+100,000", "Concurrent users")
+
+    +wrap-light
+      +wrap-subcontainer("$1,500M+", "Total value locked")
+      +wrap-subcontainer("$900M+", "Market Cap")
```
Comment (zolotokrylin) [inline src/routes/(pages)/template.pug:23]:
```suggestion
      +wrap-subcontainer("$900M+", "Combined Market Cap")
```

**Comment by matthewvogas** [comment]
> 1. is the mobile version missing in this PR?
It is not missing, the mobile version is there.
image

> 2\. where did the BTC
In the new design, it is not included (https://www.figma.com/design/39MGGVIGQ1bL3FgVzuXEdC/Holdex?node-id=7355-63376&t=ECCLRW4yFOpoMApe-4)

> 2\. Web3 is not just a financial movement; it’s an evolution of the internet.
In the new design, it is not included either, also it's not anywhere on figma, it only appears on the website.

**Comment by matthewvogas** [comment]
log: resolving comments
eta: 30m

**Comment by matthewvogas** [comment]
log: getting problems with local git - already fixed

File: src/routes/(pages)/index-mixins.pug [hunk 5]
```diff
@@ -0,0 +1,64 @@
+
+// main section
+mixin main-section
+  section.main(class="flex flex-col items-center pt-16 xs:pt- gap-19")
```
[the remark this answers is not in the document]
Comment (matthewvogas) [reply src/routes/(pages)/index-mixins.pug:4]: log: solving

File: src/routes/(pages)/index-mixins.pug [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+    block
+
+// hero section
+mixin section-hero-wrapper
+  .section-hero-wrapper(class="flex flex-col items-center justify-center bg-gray-100 gap-8 max-w-[1028px] w-full xs:px-4")
+    block
+
+// KPIs section
+mixin section-kpis-wrapper
+  .section-hero-wrapper(class="smDown:px-4 flex flex-col items-center justify-center bg-gray-100 gap-8 max-w-[1028px] w-full xs:px-4")
```
[the remark this answers is not in the document]
Comment (matthewvogas) [reply src/routes/(pages)/index-mixins.pug:14]: log: checking

**Comment by matthewvogas** [comment]
log: solving comments:  eta 15m

File: src/routes/(pages)/index-mixins.pug [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+mixin content-wrapper
+  .content-wrapper(class="flex items-start flex-col gap-8 xs:gap-4 max-w-[604px] w-full xs:px-4")&attributes(attributes)
+    block
+
+mixin wrap-section(isDark, backgroundStyle, bgClass, gradientStyle, subcontainerGradientClass)
+  section(class=(isDark ? "hidden dark:block" : "block dark:hidden") + " rounded-base-radius bg-gradient-to-r" style=backgroundStyle)
+    div(class="h-full rounded-base-radius " + bgClass + " p-2 gap-4")
+      div(class="rounded-underbase bg-gradient-to-r py-10" style=gradientStyle)
+        div(class="h-full rounded-xl " + bgClass + " gap-4")
+          div(class="h-full rounded-xl " + subcontainerGradientClass + " px-12 py-10 gap-4 xs-up:grid xs-up:grid-cols-1 sm-up:grid sm-up:grid-cols-3 md-up:grid-cols-3 lg-up:grid-cols-3 xl-up:grid-cols-3")
```
[the remark this answers is not in the document]
Comment (matthewvogas) [reply src/routes/(pages)/index-mixins.pug:45]: By doing this is forced to always be in 1st column even on web.

File: src/routes/(pages)/index-mixins.pug [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+mixin companiesStrip
+  .companiesStrip(
+    class="flex items-center justify-center smDown:justify-start smDown:items-start max-w-[1028px] tb:max-w-[794px] smDown:max-w-[604px] xs:px-4 w-full"
+  )&attributes(attributes)
+    block
+
+mixin companyLogoStripIndex(logos)
+  if logos !== undefined
+    .company-logo-wrapper(
+      class="mt-2 w-full flex flex-wrap gap-6 justify-around align-middle"
```
[the remark this answers is not in the document]
Comment (matthewvogas) [reply src/routes/(pages)/index-mixins.pug:26]: we also have custom align-middle

**Comment by markholdex** [comment]
@matthewvogas are you planning to resolve the CodeRabbit comments?

**Comment by markholdex** [comment]
> > @matthewvogas are you planning to resolve the CodeRabbit comments?
> 
> only the ones with the XSS problem, since the other 2 use custom tailwind

@matthewvogas please mark as resolved in GitHub once you resolve them. Thanks!

**Comment by matthewvogas** [comment]
solving comments eta: 30m

File: src/routes/(pages)/+page.svelte [hunk 9]
```diff
@@ -14,6 +13,19 @@
...
   $: message = Parser.parse(storeData?.message as Message);
+
+  function sanitizeStyle(style: string): string {
+    const allowedStyles = /^(background-image|padding):[\w\s,.#%-]+;$/;
+    return allowedStyles.test(style) ? style : '';
+  }
+
+  $: backgroundStyle = sanitizeStyle(
+    'background-image: linear-gradient(176deg, #1C202D 20%, #10141F 30%); padding: 0.0625rem;'
+  );
```
[the remark this answers is not in the document]
Comment (matthewvogas) [reply src/routes/(pages)/+page.svelte:24]: Tried custom tailwind approach, but not resolved.

**Comment by markholdex** [comment]
> @markholdex @matthewvogas I suggest we finalize this PR, but I have a few questions:
> 
> 1. is the mobile version missing in this PR?
> 2. where did the BTC chart and `Web3 is not just a financial movement; it’s an evolution of the internet.` quote go?

2. The BTC chart and the quote are only available on production version.

**Review by markholdex — APPROVED** [review]

**Pull request merged**